apps/right-to-rent-check/acceptance/features/tenant-details-backlink.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const _ = require('lodash');
+const steps = require('../../');
+
+Feature('Tenant Add Another Page Back Link');
+
+Before((
+  I,
+  tenantAddAnotherPage
+) => {
+  I.visitPage(tenantAddAnotherPage, steps);
+});
+
+Scenario('I am taken to the tenant-additonal-details page', (
+  I,
+  tenantAddAnotherPage,
+  tenantAdditionalDetailsPage
+) => {
+  I.click(tenantAddAnotherPage.selectors.backLink);
+  I.seeInCurrentUrl(tenantAdditionalDetailsPage.url);
```

Comment:
What is the aim of this test with the backlink? is it because the looping has previously messed up?  I see the value of adding an entry, then going back, submitting and checking the values are still present but otherwise, I'm not sure :thinking: does that mean we need to check the backlink for all our features